Libraries/SignalingClient/src/peer_connection_client.cpp

```diff
@@ -641,15 +641,18 @@ void PeerConnectionClient::OnHangingGetRead(rtc::AsyncSocket* socket)
 				bool connected = false;
 				if (ParseEntry(notification_data_.substr(pos), &name, &id, &connected))
 				{
-					if (connected)
+					if (id != my_id_)
```

Comment:
Any idea why we don't see this issue when using the native signaling server for single peer?

Reply:
yes. different code. 😋 basically the new signaling implementation is not 100% the same as the native one.